### ISSUE TITLE
support webpack v5

### ISSUE THREAD

**Issue by aladdin-add** [merkle-open/webpack-config-plugins#55]

**Comment by janbiasi** [comment]
Hey @aladdin-add, thanks for submitting. We're actually discussing internally how we will solve the requirement to make the plugins compatible with multiple webpack versions 😃We're coming back to this as soon as we find a solution!

**Comment by janbiasi** [comment]
We decided to stay a little bit longer with Webpack 4, as version 5 isn't fully implemented nor the milestone of the first release was reached (see their [milestones](https://github.com/webpack/webpack/milestone/18)), I'll close this for now and we'll come back when webpack 5 gets stable 😃

**Comment by heavenshell** [comment]
Hi, webpack5 stable was released, any news about this?

**Comment by jantimon** [comment]
absolutely we should try to find some time to work on the next major version 👍

**Comment by nemchik** [comment]
Duplicate https://github.com/namics/webpack-config-plugins/issues/68
This issue seems to have more active discussion, so maybe keep this and close the other?

**Comment by yunwuxin** [comment]
@janbiasi

**Comment by nemchik** [comment]
At this point I've dug into the makeup of these packages and made up a webpack config myself. The dependencies end up being significantly slimmed down (about cut in half) and it works with webpack 5.

The majority of what I did was based on https://github.com/microsoft/TypeScript-Babel-Starter which handles js and ts. Aside from that I've got the mini css plugin and sass loader. I haven't done the other asset stuff yet. I'm not planning to release anything, just outlining what I did so others can reproduce.
